lang/en/texts/android.html

```diff
@@ -0,0 +1,30 @@
+<h4>Mobile</h4>
+<a download="4. Additives - Mobile - EN.png" href="https://world.openfoodfacts.org/files/presskit/PressKit/android/screenshots/4. Additives - Mobile - EN.png" target="_blank"><img width="300px" src="https://world.openfoodfacts.org/files/presskit/PressKit/android/screenshots/4. Additives - Mobile - EN.png"></a>
```

Comment:
For [accessibility](https://www.w3.org/TR/html51/semantics-embedded-content.html#alt-text), all of the `img` tags really should have `alt` texts that describe them, ie. `alt="Screenshot of the OpenFoodFacts Android App - Safety and exposure assessment: Identify food additives your risk of overconsuming"` [sic].